### PR TITLE
Display empty string when no gender specified

### PR DESCRIPTION
**Pull request by Pujitha97**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Gender.java [hunk 1]
```diff
@@ -77,7 +82,7 @@ public static boolean isValidGender(String test) {
 
     @Override
     public String toString() {
-        return value.toString();
+        return IsGenderSet ? value.toString() : "";
     }
```
Comment (OscarWang114) [inline src/main/java/seedu/address/model/person/Gender.java]: I think the title of this PR should be 'display "empty string" when no gender specified'?

**Comment by OscarWang114** [comment]
Overall looks good to me except for the PR title because `null` can never be displayed unless it's a `String` that is assigned with the four characters "null", which isn't helpful in any way. Help me clarify this point and then I'll merge this PR.

File: src/main/java/seedu/address/model/person/Gender.java (hunk 1 above)
Comment (Juxarius) [reply src/main/java/seedu/address/model/person/Gender.java]: tbh the capital I is glaring strongly at me also, WHY ISNT TRAVIS BITCHING ABOUT THIS?

Comment (OscarWang114) [reply src/main/java/seedu/address/model/person/Gender.java]: Nice observation.

**Comment by Pujitha97** [comment]
@OscarWang114 When no gender is specified , instead of displaying NOT_SPECIFIED , just like all the other fields , this displays an empty string: "".
Let me know if you have further concerns!

**Review by OscarWang114 — APPROVED** [review]
Approved

**Pull request merged**